.github/workflows/js-and-android-test.yml

```diff
@@ -2,7 +2,7 @@ name: Test pushes and PRs
 on:
   push:
   pull_request:
-    types: [assigned, opened, ready_for_review]
+    types: [assigned, opened, ready_for_review, closed]
```

Comment:
This is about the PR being closed, not necessarily being merged. Also is the PR automatically closed when merged?

Maybe we should use the merge event instead `github.event.pull_request.merged`?

Reply:
I'm afraid that, while that's an attribute in the github object within the github action, it's not a `type` that can be used in the github action trigger, as indicated [here](https://stackoverflow.com/a/67833464/891440). We can add an additional filter in the job to not run if it's closed, but it's added complexity and it really does not matter if we run something when closing a PR.